examples/ensemble/plot_stack_predictors.py

```diff
@@ -76,7 +76,7 @@ def load_ames_housing():
     X, y = shuffle(X, y, random_state=0)
 
     X = X[:600]
-    y = y[:600]
+    y = y.iloc[:600]
```

Comment:
We should probably be more explicit here. I am not sure that everyone knows what happens when slicing a dataframe with integers :)

Let's do the following to be extra explicit:

```python
    X = X.loc[:, features]
    X, y = shuffle(X, y, random_state=0)

    X = X.iloc[:600]
    y = y.iloc[:600]
```